CHANGELOG.md

```diff
@@ -1,4 +1,58 @@
-0.16.0 (2022-12-006)
+0.17.0 (2023-02-17)
+===
+
+Plan
+---
+* Fix a bug where `SemiSpace::get_available_pages` returned unused pages as 'available pages'. It should return half of the unused pages as 'available'.
+* Fix a bug where generational plans may make full-heap GC decision before all the releasing work is done. Now plans have a `Plan::end_of_gc()` method
+  that is executed after all the GC work is done.
+* Fix a bug where generational plans tended to trigger full heap GCs when heap is full. Now we properly use Appel-style nursery and allow
+  the nursery to extend to the heap size.
+* Add a feature `immix_zero_on_release` as a debug feature for Immix to eagerly zero any reclaimed memory.
+* Fix a bug where the alloc bits for dead objects were still set after the objects were reclaimed. Now Immix eagerly clears the alloc bits
+  once the memory is reclaimed.
+* Fix a bug where Immix uses forwarding bits and forwarding pointers but did not declare it.
+
+
+Allocator
+---
+* Fix a bug about the maximum object size in `FreeListAllocator`. The allowed maximum object size depends on both the block size and the largest bin size.
+
+
+Scheduler
+---
+* Add a 'sentinel' mechanism to execute a specified 'sentinel' work packet when a bucket is drained, and prevent the GC from entering the next stage depending
+  on the sentinel packet. This makes it possible to expand the transitive closure multiple times. It replaces `GCWorkScheduler::closure_end()`.
+
+
+API
+---
+* Add a new set of APIs to support binding-specific weak reference processing:
+  * Add `Scanning::process_weak_refs()` and `Scanning::forward_weak_refs()`. They both supply an `ObjectTracerContext` argument for retaining and
+    updating weak references. `Scanning::process_weak_refs()` allows a boolean return value to indicate if the method should be called again
+    by MMTk after finishing transitive closures for the weak references, which can be used to implement ephemeral weak references.
```

Comment:
The return value here is intended to support "ephemeron".  It is a well-known term, and it is used in V8.  I can't find any use of the term "ephemeral weak reference".

```suggestion
    by MMTk after finishing transitive closures for the weak references, which can be used to implement ephemerons.
```

Reply:
Done. Thanks.